clang/test/CheckedCRewriter/b10_allsafepointerstruct.c

```diff
@@ -40,7 +49,15 @@ struct p *foo() {
   struct p *z = (struct p *) sus(x, y);
   return z;
 }
-//CHECK: _Ptr<struct p> foo(void) {
+//CHECK_NOALL: _Ptr<struct p> foo(void) {
+//CHECK_NOALL:  _Ptr<struct p> x = ((void *)0);
+//CHECK_NOALL: _Ptr<struct p> y = ((void *)0);
+//CHECK_NOALL:   _Ptr<struct p> z =  (struct p *) sus(x, y);
+//CHECK_ALL: _Ptr<struct p> foo(void) {
```

Comment:
Have you confirmed that all the extra `CHECK_NOALL`s are reasonable, i.e., that the tool is doing the right thing?

Reply:
Nope, but I'm going to be revisiting this batch of tests in order to add checked annotations to the structs, so I can take a look to make sure the output makes sense.

Reply:
Same comment as above about the annotation location.

If this is going to be a big pain to do, then we could put it off as an enhancement and just take things as they are.